setup.py

```diff
@@ -47,6 +48,8 @@
         'test': [
           'plone.app.testing',
           'plone.app.dexterity',
+          'Products.ATContentTypes',
+          'unittest2',
```

Comment:
We still have envs in our travis matrix for Plone 4.2 and 4.1 with Python 2.6 support.  I'm not okay with dropping that unless we are saying explicitly that we no longer support 4.2.